x-pack/filebeat/docs/inputs/input-http-endpoint.asciidoc

```diff
@@ -255,6 +268,10 @@ For example, `["content-type"]` will become `["Content-Type"]` when the filebeat
 
 This option copies the raw unmodified body of the incoming request to the event.original field as a string before sending the event to Elasticsearch.
 
+[float]
+==== `crc.provider`
+
+This option defines the provider of the webhook that uses CRC (Challenge-Response Check) for validating the endpoint. The HTTP endpoint input is responsible for ensuring the authenticity of incoming webhook requests by generating and verifying a unique token. By specifying the `crc.provider`, you ensure that the system correctly handles the specific CRC validation process required by the chosen provider.
```

Comment:
Question: Should we mention that `secret.value` is required when `crc.provider` is configured as it seems it is validated [here](https://github.com/elastic/beats/pull/35204/files#diff-72430a3a126776d3059fc756070316d0c7a70e799c250cbac4193dd6126c4489R95)

Reply:
Yes, this should be noted here.

Reply:
Yes, actually after further review with @P1llus we will take a slightly different approach on a new PR to better handle this relationship between the CRC and the secret. I'll let you know once it is ready. Thank you for pointing this out!

Reply:
Thanks. Should I abandon #35617?

Reply:
Yes, please. I am adding a new setting `crc.secret` since, in the case of Zoom at least, they are two different tokens. Thank you!

Reply:
@efd6 @bhapas here you have the PRs with the latest changes about this (https://github.com/elastic/beats/pull/35604   and https://github.com/elastic/integrations/pull/6411) just in case you want to take a look at them. Thanks!